### ISSUE TITLE
Uncaught Error: Position (37.666666666666664, Infinity) is less than the current token's start ((

### ISSUE THREAD

**Issue by rtack** [atom/atom#13223]
0. In Split window view. 
1. Having markdown file open in right pane. with unsaved changes.
2. Using CTRL+SHIFT+M to open preview. 
3. moving the markdown file to the left pane by dragging with the mouse.

**Atom Version**: 1.13.0-beta2
**Electron Version**: 1.3.6
**System**: Linux Mint
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Position (37.666666666666664, Infinity) is less than the current token's start ((38, 0))

```
At /usr/share/atom-beta/resources/app.asar/node_modules/display-index/dist/token-iterator.js:251

Error: Position (37.666666666666664, Infinity) is less than the current token's start ((38, 0))
    at TokenIterator.translateScreenPosition (/usr/share/atom-beta/resources/app.asar/node_modules/display-index/dist/token-iterator.js:251:15)
    at DisplayLayer.module.exports.DisplayLayer.translateScreenPosition (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/display-layer.js:1491:52)
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.translateToBufferMarkerLayerFindParams (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:220:46)
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.findMarkers (/usr/share/atom-beta/resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:133:21)
    at DecorationManager.module.exports.DecorationManager.decorationsStateForScreenRowRange (/usr/share/atom-beta/resources/app.asar/src/decoration-manager.js:151:21)
    at TextEditor.module.exports.TextEditor.decorationsStateForScreenRowRange (/usr/share/atom-beta/resources/app.asar/src/text-editor.js:1716:37)
    at TextEditorPresenter.module.exports.TextEditorPresenter.fetchDecorations (/usr/share/atom-beta/resources/app.asar/src/text-editor-presenter.js:1378:44)
    at TextEditorPresenter.module.exports.TextEditorPresenter.getPreMeasurementState (/usr/share/atom-beta/resources/app.asar/src/text-editor-presenter.js:125:14)
    at TextEditorComponent.module.exports.TextEditorComponent.updateSyncPreMeasurement (/usr/share/atom-beta/resources/app.asar/src/text-editor-component.js:264:60)
    at TextEditorComponent.module.exports.TextEditorComponent.updateSync (/usr/share/atom-beta/resources/app.asar/src/text-editor-component.js:196:12)
    at /usr/share/atom-beta/resources/app.asar/src/text-editor-component.js:323:28
    at ViewRegistry.module.exports.ViewRegistry.performDocumentUpdate (/usr/share/atom-beta/resources/app.asar/src/view-registry.js:231:9)
    at /usr/share/atom-beta/resources/app.asar/src/view-registry.js:3:59
```

### Commands

```
     -9:38.7.0 editor:newline (input.hidden-input)
     -9:36.3.0 editor:move-to-end-of-screen-line (input.hidden-input)
     -9:34.6.0 core:backspace (input.hidden-input)
     -9:26.1.0 editor:move-to-end-of-screen-line (input.hidden-input)
  3x -9:14.8.0 core:move-down (input.hidden-input)
  2x -9:11.7.0 editor:delete-line (input.hidden-input)
     -9:10.8.0 core:save (input.hidden-input)
  2x -9:10.3.0 core:move-up (input.hidden-input)
     -9:09.7.0 editor:move-to-end-of-screen-line (input.hidden-input)
     -9:08.8.0 core:save (input.hidden-input)
     -9:07.6.0 core:move-down (input.hidden-input)
     -9:01.2.0 core:save (input.hidden-input)
     -0:41.2.0 git-plus:checkout (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:09.2.0 editor:newline (input.hidden-input)
     -0:05.3.0 markdown-preview:toggle (input.hidden-input)
     -0:00.1.0 tree-view:toggle (input.hidden-input)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "linter-rubocop"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "node_modules",
      "public"
    ],
    "openEmptyEditorOnStart": false,
    "telemetryConsent": "limited"
  },
  "editor": {
    "fontSize": 16,
    "invisibles": {},
    "preferredLineLength": 120,
    "scrollPastEnd": true,
    "showIndentGuide": true,
    "showInvisibles": true,
    "softWrap": true,
    "softWrapAtPreferredLineLength": true
  }
}
```

### Installed Packages

```coffee
# User
asciidoc-assistant, v0.2.1 (active)
asciidoc-image-helper, v1.0.0 (active)
asciidoc-preview, v2.6.0 (active)
asciidoctor-preview, v0.3.1 (active)
atom-beautify, v0.29.13 (active)
autocomplete-asciidoc, v0.1.1 (active)
flatten-json, v0.1.4 (inactive)
font-awesome-snippetset, v0.0.6 (active)
git-plus, v5.25.1 (active)
highlight-line, v0.11.1 (active)
highlight-selected, v0.11.2 (active)
json-converter, v0.2.4 (inactive)
language-asciidoc, v1.5.3 (active)
language-csv, v1.1.2 (active)
language-cypher, v0.3.0 (active)
language-generic-config, v1.2.0 (active)
language-scala, v1.1.8 (active)
language-slim, v0.5.0 (active)
linter, v1.11.19 (active)
linter-rubocop, v0.5.0 (inactive)
linter-ruby, v1.2.2 (active)
linter-scss-lint, v3.0.4 (active)
markdown-pdf, v1.5.0 (inactive)
maximize-panes, v0.2.0 (inactive)
php-cs-fixer, v2.5.9 (active)
pigments, v0.38.0 (active)
pretty-json, v1.6.1 (active)
project-manager, v3.3.3 (active)
ruby-block, v0.3.5 (active)
ruby-slim, v0.2.0 (active)
Sublime-Style-Column-Selection, v1.7.2 (active)
tablr, v1.6.1 (active)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (active)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (active)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.2 (active)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.1 (active)
metrics, v1.0.0 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.12.1 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.46.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by Ben3eeE** [comment]
1. Open markdown file and split right
1. Soft wrap is not behavior correctly, it is flashing
1. Resize pane and exception appears

![soft wrap qqqq](https://cloud.githubusercontent.com/assets/1058982/20271484/2df35cc8-aa8b-11e6-9cb1-cdbc4c102ee2.gif)

**Comment by maxbrunsfeld** [comment]
I'm sorry; my previous fix for this issue didn't get shipped properly. I believe this is fixed in 1.13.0-beta3. Thanks for the report!
